aquadoggo/src/graphql/query.rs

```diff
@@ -8,7 +8,14 @@ use crate::errors::Result;
 
 #[derive(SimpleObject)]
 pub struct BookmarkDocument {
+    /// The document's ID
     document: String,
+
+    /// Comma-separated hashes of the document graph's tips, which are required for posting update
+    /// or delete operations for this document.
+    previous_operations: String,
```

Comment:
The alternative here would be actually storing foreign keys of the operations that are being referred to here in a `previous_operations` table.